23-07-workshop/finished/Leavitts_Footsteps.ipynb

```diff
@@ -0,0 +1,1937 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Being slow, commented out for now.
</pre><p> </p><p>I think this was a temporary issue on the MAST side. Seems to be back up to speed now</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='79'/>